src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -499,14 +517,14 @@ export class Tv2SplitScreenActionFactory {
   }
 
   private createInsertLastVideoClipToInputAction(blueprintConfiguration: Tv2BlueprintConfiguration, inputIndex: number, audioMode: Tv2AudioMode): Tv2SplitScreenInsertLastVideoClipInputAction {
-    const audioTimelineObjects: TimelineObject[] = this.audioTimelineObjectFactory.createVideoClipAudioTimelineObjects(blueprintConfiguration, {
+    const audioTimelineObjects: Tv2BlueprintTimelineObject[] = this.audioTimelineObjectFactory.createVideoClipAudioTimelineObjects(blueprintConfiguration, {
       fileName: inputIndex,
       audioMode
     } as unknown as Tv2VideoClipManifestData)
 
     return {
-      id: `insert_last_video_clip_to_split_screen_input_${inputIndex}${audioMode ? '_vo' : ''}_action`,
-      name: `Insert last Video ${audioMode ? ' Voice Over ' : ''} Clip in DVE input ${inputIndex}`,
+      id: `insert_last_video_clip_to_split_screen_input_${inputIndex}${audioMode === Tv2AudioMode.VOICE_OVER ? '_vo' : ''}_action`,
+      name: `Insert last Video ${audioMode === Tv2AudioMode.VOICE_OVER ? ' Voice Over ' : ''} Clip in DVE input ${inputIndex}`,
       description: 'Insert last Video Clip in DVE input ${inputIndex}',
```

Comment:
Not using backticks, when it should.

Reply:
Can you elaborate? What should be using backticks that isn't using it?

Reply:
The string should use backticks, so the `inputIndex` is put into the string as defined.
Currently the output string literally says `Insert last Video Clip in DVE input ${inputIndex}`

Reply:
Ahh, I was looking a line 527 since that was the one with changes. I see it now. Good catch.
It has been updated